sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClientBuilder.java

```diff
@@ -46,6 +46,14 @@
  * {@link #credential(TokenCredential) credential(TokenCredential)} give the builder access credential.
  * </p>
  *
+ * <p><strong>Instantiating an asynchronous Text Analytics Client</strong></p>
+ *
+ * {@codesnippet com.azure.ai.textanalytics.TextAnalyticsAsyncClient.instantiation}
+ *
+ * <p><strong>Instantiating a synchronous Text Analytics Client</strong></p>
```

Comment:
twice?

Reply:
one is for Async and another is for Sync. By comparing the length, you will see it intuitively.